src/clients/python/schemas/WebViewOptions.py

```diff
@@ -0,0 +1,69 @@
+# DO NOT EDIT: This file is auto-generated by generate-schema/index.ts
+from enum import Enum
+from typing import Optional, Union
+import msgspec
+
+class WebViewContentUrl(msgspec.Struct, kw_only=True, omit_defaults=True): 
+    url: str
+    """Url to load in the webview. Note: Don't use data URLs here, as they are not supported. Use the `html` field instead.""" 
+    headers: Optional[dict[str, str]] = None
```

Comment:
Matter of preference, but I prefer `Union[T, None]` over `Optional[T]`. Some [prior discussion](https://fastapi.tiangolo.com/python-types/#using-union-or-optional).

Reply:
That's good to know!